jekyll/_cci2/pipelines.adoc

```diff
@@ -0,0 +1,138 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+---
+= Pipelines overview
+:page-description: This document introduces the concept of pipelines and shows how pipelines can be triggered and what they include.
+:experimental:
+:icons: font
+:page-layout: classic-docs
+:page-liquid:
+
+CircleCI pipelines are the highest-level unit of work, encompassing a project's full configuration (for example, `.circleci/config.yml`, or multiple configuration files if that is how your project is set up). Pipelines coordinate the execution of workflows and jobs to build, test, and deploy your code. Workflows can be conditional, based on xref:workflows#using-contexts-and-filtering-in-your-workflows[filters], xref:workflows#workflows-configuration-examples[job requirements], and xref:selecting-a-workflow-to-run-using-pipeline-parameters#[pipeline parameters].
```

Comment:
How about something like: `CircleCI pipelines are the highest-level unit of work, encompassing all work configured and associated with the pipeline trigger` 😬  ?

Reply:
It's a bit delicate because it should encompass both classic and standalone.
How about
> CircleCI Pipelines are the highest-level unit of work in a project, and they are described by a configuration file (for example, `circleci/config.yml`. Triggers can be setup to automate when a pipeline will be run.